doc/user/content/guides/cdc-mysql.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Change Data Capture (MySQL)"
+description: "How to create real-time materialized views from MySQL changelog data with Materialize."
+weight: 10
+menu:
+  main:
+    parent: guides
+---
+
+Change Data Capture (CDC) allows you to track and propagate changes in a MySQL database to downstream consumers based on its binary log (`binlog`). In this guide, we'll cover how to use Materialize to create and efficiently maintain real-time materialized views on top of CDC data.
+
+## Kafka + Debezium
```

Comment:
Since we don't have another h2 section on this page, I would just get rid of the heading and make all the h3s h2s.

Reply:
I was half-preparing for when we have a direct MySQL source in the future.

Reply:
Makes sense.